### ISSUE TITLE
Guild achievement bugged.

### ISSUE THREAD

**Issue by BrotherWolf7** [gamefreedomgit/Maelstrom#4075]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**

The guild achievement [Working as a Team] is bugged for my guild. We have several  characters with maxed out professions, but some of those professions are not counting toward the completion of the achievement. For instance, the guild leader has capped skinning and leatherworking, but skinning has not been checked off in the achievement. Other professions include alchemy, jewelcrafting, mining, tailoring, enchanting and inscription. Each of the characters with the professions have reached the reputation of Honored and have completely capped their profession skills. 

**How to reproduce:**

**How it should work:**

When a character in the guild obtains 525 skill points in a profession with a reputation of Honored or greater, the achievement should check off the professions and they should count toward the completion of the achievement.

**Database links:**

![World of Warcraft 7_31_2023 12_15_08 AM](https://github.com/gamefreedomgit/Maelstrom/assets/140998583/7a58fb16-4d69-4a0b-b73d-c7f4098f912e)
![World of Warcraft 7_31_2023 12_15_24 AM](https://github.com/gamefreedomgit/Maelstrom/assets/140998583/2f9fabc3-4c8a-48fb-a9cb-6a00332bdb71)

**Comment by UltraNix** [comment]
@BrotherWolf7 I believe they capped their professions while being already in your guild?

**Comment by BrotherWolf7** [comment]
Yes. The professions were leveled and capped by people who are in the guild. I am the guild leader. Notice how skinning and leatherworking are both capped on my character. I am the ONLY leatherworker and the ONLY skinner in my guild. The leatherworking part of the achievement is green. The skinning achievement is still grayed out.

**Comment by UltraNix** [comment]
I see skinning is 540. Why is that?
What's your skill in mining?

**Comment by BrotherWolf7** [comment]
@UltraNix Worgen's racial passive gives them +15 to skinning, so 525+15=540. The same thing occurs with the draenei's jewelcrafting. They can have a jewelcrafting skill of maximum 540. The paladin is the miner, and his mining skill is 525.
